start.yaml

```diff
@@ -1,12 +1,16 @@
 cheat: 0
 health: 10
+armor protection: .95
 held item: Sword
 inventory:
 - Sword
 - Healing Potion
-max health: 10
+- Leather Jacket
+- Leather Boots
+- Old Pants
```

Comment:
Maybe you picked these up before the game starts, while you chase your friend into the dungeons.

Reply:
Sorry, I actually added locally that but I don't know why it wasn't commited.